doc/admin_doc/install_doc/installation.rst

```diff
@@ -57,13 +57,13 @@ Or, for Fedora 32+:
     # yum install -y git gcc libtidy python3 python3-devel python3-pip zeromq-devel dnf-plugins-core
     # dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
     # dnf install -y docker-ce docker-ce-cli
-    # cat <<EOF > /etc/yum.repos.d/mongodb-org-4.4.repo
-    [mongodb-org-4.4]
+    # cat <<EOF > /etc/yum.repos.d/mongodb-org-7.0.repo
+    [mongodb-org-7.0]
     name=MongoDB Repository
-    baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/4.4/x86_64/
+    baseurl=https://repo.mongodb.org/yum/redhat/9Server/mongodb-org/7.0/x86_64/
```

Comment:
If you specify `9Server` as distro version, update the above mentioned OS versions. You can also use tabs (see the run file documentation pages) for several versions. You can remove `CentOS 7` and replace it by `RHEL/Rocky/Alma Linux 8+` as version 8 is still supported.